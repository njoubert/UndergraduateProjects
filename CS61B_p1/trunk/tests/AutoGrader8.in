The athlete lives around here.
The beerdrinker lives in the bavarian house.
Peter is not the athlete.
Tomas lives around here.
Tomas lives in the czech house.
Alex lives in the american house.
What do you know about Peter?
What do you know about Tomas?
What does Alex do?

