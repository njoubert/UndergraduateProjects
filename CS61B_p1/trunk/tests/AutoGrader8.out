1. The athlete lives around here.
2. The beerdrinker lives in the bavarian house.
3. Peter is not the athlete.
4. Tomas lives around here.
5. Tomas lives in the czech house.
6. Alex lives in the american house.
Q: What do you know about Peter?
A: Peter is the beerdrinker and lives in the bavarian house.
Q: What do you know about Tomas?
A: Tomas lives in the czech house.
Q: What does Alex do?
A: I don't know.
